src/main/iidxhook-util/d3d9.c

```diff
@@ -637,47 +637,45 @@ iidxhook_util_d3d9_iidx11_to_17_fix_uvs_bg_videos(struct hook_d3d9_irp *irp)
                  (vertices[1].x >= 472.0f && vertices[1].x <= 476.0f &&
                   iidxhook_util_d3d9_float_equal(vertices[1].y, 0.0f, 0.1f)) &&
                  (vertices[2].x >= 472.0f && vertices[2].x <= 476.0f &&
-                  iidxhook_util_d3d9_float_equal(
-                      vertices[2].y, 416.0f, 0.1f)) &&
+                  vertices[2].y >= 415.0f && vertices[2].y <= 416.0f) &&
                  (vertices[3].x >= 164.0f && vertices[3].x <= 168.0f &&
-                  iidxhook_util_d3d9_float_equal(
-                      vertices[3].y, 416.0f, 0.1f))) ||
+                  vertices[3].y >= 415.0f && vertices[3].y <= 416.0f)) ||
                 /* double top left */
-                ((iidxhook_util_d3d9_float_equal(vertices[0].x, 6.0f, 0.1f) &&
+                ((vertices[0].x >= 6.0f && vertices[0].x <= 11.0f &&
```

Comment:
Seeing in your PR description that you have the exact numbers, I would use these instead of ranges. That makes the heuristic more specific which is preferred if you have exact numbers. The larger the rage, the higher the "risk" that this might be applied to another texture as well.

Reply:
Alright, that if statement is breaking my brain but I tested it on 9th, 10th, and newer styles and everything seems to be working fine now.